.github/ISSUE_TEMPLATE/4-report-a-flaky-test.yml

```diff
@@ -19,11 +19,24 @@ body:
       placeholder: e.g. `test-fs-stat-bigint`
     validations:
       required: true
-  - type: input
+  - type: dropdown
     attributes:
       label: Platform
       description: The platform the test is flaky on.
-      placeholder: e.g. `macos` or `linux`
+      multiple: true
+      options:
+        - AIX
+        - FreeBSD
+        - Linux ARM64
+        - Linux ARMv7
+        - Linux PPC64LE
+        - Linux s390x
+        - Linux x64
+        - Other
+        - SmartOS
+        - Windows
+        - macOS ARM64
+        - macOS x64
```

Comment:
```suggestion
        - AIX
        - FreeBSD
        - Linux ARM64
        - Linux ARMv7
        - Linux PPC64LE
        - Linux s390x
        - Linux x64
        - macOS ARM64
        - macOS x64
        - Other
        - SmartOS
        - Windows
```

Reply:
Thanks, the vim `:'<,'>sort` didn't work well. I've included the `Other` as last option.